German translations and validations 3

rebplu opened the pull request.

rebplu committed:
Update organization.ttl

Review German translations and add Archive

rebplu committed:
Update philosophies.ttl

German translations

rebplu committed:
Update physical-entity.ttl

Review and add German translations. I would recommend Place = Ort and not Platz. Platz is understood mostly as a square.

rebplu committed:
Update physical-geography.ttl

German translations

rebplu committed:
Update prosodic-structure.ttl

Review German translations

rebplu committed:
Update publishing.ttl

Review German translations

rebplu committed:
Update scholarly-editing.ttl

German translations untill ApparatusEntry

rebplu committed:
Update document.ttl

Ausdruck = Expression

hanscools commented on Nie-ontologies/Generic-ontologies/organization.ttl:
```diff
@@ -43,14 +43,14 @@ org:Role
 	a rdfs:Class;
-	rdfs:label "organization creator"@en, "Organisation-Erschafferin"@de;
+	rdfs:label "creator role of organization"@en, "Erstellerrolle einer Organisation"@de;
 	rdfs:comment """Publisher as role of an organization."""@en;
```
Publisher -> Creator

rebplu replied:
Ok.

rebplu replied:
Commit aa34d6e

hanscools commented on Nie-ontologies/Generic-ontologies/organization.ttl:
```diff
@@ -89,7 +89,7 @@ org:Politician
 	a rdfs:Class;
 	#rdfs:isDefinedBy <http://e-editiones.ch/ontology/organization#>;
-	rdfs:label "politician role"@en, "Politiker-Rolle"@de;
+	rdfs:label "politician role"@en, "Rolle eines Politikers"@de;
```
politician role -> politician role of person; Rolle eines Politikers -> Politikerrolle einer Person

rebplu replied:
Ok.

rebplu replied:
Commit aa34d6e

hanscools commented on Nie-ontologies/Generic-ontologies/publishing.ttl:
```diff
@@ -228,22 +228,22 @@ publish:LastPublication
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/publishing#>;
 	a rdfs:Class;
-	rdfs:label "serial volume"@en, "Reihe-Ausgabevolumen"@de, "volume série"@fr, "volume seriale"@it;
+	rdfs:label "serial volume"@en, "Serienband"@de, "volume série"@fr, "volume seriale"@it;
```
To be sure: 'Serienband' is a set of serial issues?

rebplu replied:
Yes I'm sure.
In Publishing-Context: 
volume = Band
issue = Ausgabe
set of serial issues = Reihe von Serienausgaben = Serienband = serial volume

hanscools commented on Nie-ontologies/Generic-ontologies/publishing.ttl:
```diff
@@ -332,7 +332,7 @@ publish:hasPublishingState
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/publishing#>;
 	a owl:ObjectProperty;
-	rdfs:label "has publishing"@en, "hat Herausgeben"@de, "ha editoria"@it; #""@fr; ?  
+	rdfs:label "has publishing"@en, "hat Herausgeber"@de, "ha editoria"@it; #""@fr; ?  
```
The object is a procedure, not an agent.

rebplu replied:
Ok.

rebplu replied:
Commit 47b31d8

hanscools commented on Nie-ontologies/Generic-ontologies/publishing.ttl:
```diff
@@ -358,7 +358,7 @@ publish:hasElectronicPublication
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/publishing#>;
 	a owl:ObjectProperty;
-	rdfs:label "has web publication"@en, "hat Web-Publikation"@de, "a une publication web"@fr, "ha una pubblicazione web"@it;
+	rdfs:label "has web publication"@en, "hat Onlinepublikation"@de, "a une publication web"@fr, "ha una pubblicazione web"@it;
```
Is 'Onlinepublikation' not more general than 'Web-Publikation'?

rebplu replied:
I would say it is the same: [https://de.m.wikipedia.org/wiki/Web-Publikation?title=Web-Publikation&redirect=no](https://de.m.wikipedia.org/wiki/Web-Publikation?title=Web-Publikation&redirect=n)
Normally in German one speaks of Online instead of Web. Nevertheless changed it because of consistency.

rebplu replied:
Commits 716948b and 2da6731

hanscools commented on Nie-ontologies/Generic-ontologies/publishing.ttl:
```diff
@@ -599,7 +599,7 @@ publish:hasPrinterDescriptionLiteral
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/publishing#>;
 	a owl:ObjectProperty;
-	rdfs:label "is serial of volume"@en, "ist Reihe-Ausgabe eines Volumens"@de, "est publication sérielle du volume"@fr, "è pubblicazione seriale del volume"@it;
+	rdfs:label "is serial of volume"@en, "ist Serie eines Bandes"@de, "est publication sérielle du volume"@fr, "è pubblicazione seriale del volume"@it;
```
is serial of volume -> is serial issue of volume; ist Serie eines Bandes -> ist Serienausgabe eines Bandes ?

rebplu replied:
Serienausgabe eines Bandes. Corrected it.

rebplu replied:
Commit a901c55

hanscools commented on Nie-ontologies/Generic-ontologies/scholarly-editing.ttl:
```diff
@@ -189,14 +189,14 @@ scholarly-editing:Siglum
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/scholarly-editing#>;
 	a rdfs:Class;
-	rdfs:label "editing apparatus"@en, "Edierungsapparatus"@de, "apparatus editionis"@la, "apparat éditionel"@fr, "apparato edizione"@it; 
+	rdfs:label "editing apparatus"@en, "textkritischer Apparat"@de, "apparatus editionis"@la, "apparat éditionel"@fr, "apparato edizione"@it; 
```
I'll check on the other labels.

rebplu replied:
Changed it to "editorischer Apparat" because later critical apparatus = textkritischer Apparat. Maybe we discuss that on Friday.
See Commit 9198738

rebplu committed:
Update organization.ttl

Corrections (comments Hans)

rebplu committed:
Update publishing.ttl

publishing = procedure = "Publizieren"/"Publizierungs-"

rebplu committed:
Update publishing.ttl

online = Web

rebplu committed:
Update publishing.ttl

Correction because Composita in German consistent without "-"

rebplu committed:
Update publishing.ttl

is serial of volume -> is serial issue of volume; ist Serie eines Bandes -> ist Serienausgabe eines Bandes

rebplu committed:
Update scholarly-editing.ttl

Review German Translations and change back textkritischer Apparat to editorischer Apparat.

rebplu committed:
Update scholasticism.ttl

Review German translations

rebplu commented:
Memo: check everywhere procedure = Verfahren? (Commit 9b59fec , but can be more)

rebplu committed:
Update concept.ttl

procedure = Verfahren (more precise than Vorgehen)

rebplu committed:
Update standoff-annotation.ttl

Review German translations

rebplu committed:
Update teaching.ttl

Review German translations

rebplu committed:
Update terminology-code-systems.ttl

German translations

rebplu committed:
Update terminology-code-systems.ttl

small case error missed

rebplu committed:
Update text-editing.ttl

Review German tranlsations

rebplu committed:
Update text-expression.ttl

Review German translations

rebplu committed:
Update text-structure.ttl

Review German translations

rebplu committed:
Update text.ttl

Review German tranlsations

rebplu committed:
Update time.ttl

Added labels

rebplu reviewed (commented):
time.ttl/ Commit c6a8614 
TBD:
- en-Labels always ok?
- German tranlsations still has to be finished (calculating = Berechnung?)
- rdfs:comment should be sometimes skos:note?

hanscools commented on Nie-ontologies/Generic-ontologies/standoff-annotation.ttl:
```diff
@@ -323,7 +323,7 @@ sa:TitleTag
 	a rdfs:Class;
-	rdfs:label "header stand-off tag"@en, "Kopfzeile-Stand-off-Tag"@de;
+	rdfs:label "heading stand-off tag"@en, "Überschrift-Stand-off-Tag"@de;
 	rdfs:comment """Stand-off tag annotating a text header literal."""@en;
```
header -> heading

hanscools commented on Nie-ontologies/Generic-ontologies/text-expression.ttl:
```diff
@@ -100,14 +100,14 @@ text-expression:Body
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/text-expression#>;
 	a rdfs:Class;
-	rdfs:label "text expression conclusion"@en, "Textexpressionskonklusion"@de, "conclusion d'expression de texte"@fr, "conclusione del espressione di testo"@it;
+	rdfs:label "text expression conclusion"@en, "Textexpressionszusammenfassung"@de, "conclusion d'expression de texte"@fr, "conclusione del espressione di testo"@it;
```
-zusammenfassung -?-> -fazit

hanscools commented:
> time.ttl/ Commit [c6a8614](https://github.com/nie-ine/Ontologies/commit/c6a86146523f0291ef2f514c270098aabcc93d31)
> TBD:
> 
>     * en-Labels always ok?

They have to be reviewed too.
> 
>     * German tranlsations still has to be finished (calculating = Berechnung?)
> 
>     * rdfs:comment should be sometimes skos:note?

Could be; do you have an example?
Note: also skos:example can be used.

Commit:
Merge branch 'master' into germanTranslations_and_validations_3
